packages/react-notion-x/src/block.tsx

```diff
@@ -884,6 +885,28 @@ export const Block: React.FC<BlockProps> = (props) => {
         </tr>
       )
 
+    case 'external_object_instance':
+      switch (block.format.domain) {
```

Comment:
I've found it's helpful err on the side of caution when dealing with notion data — especially across different block versions and future updates.

So for any property accessors I try to use `?.` to ensure the code doesn't throw (since if it does the user will get a blank screen).

So maybe `block.format?.domain` and `block.format?.original_url`?

Some parts of this codebase may not be following this rule, but I'd like to try to move towards this going forwards.